app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
```

Comment:
The `people` dictionary is used to store `Person` instances by name. This could lead to issues if names are not unique, as it will overwrite existing entries with the same name. Consider using a different key or structure to ensure uniqueness.